docs/components/checkbox-field/index.md

```diff
@@ -0,0 +1,158 @@
+# Checkbox Field
+
+Provides an underlying checkbox element building on top of the Field component.
+
+## Label
+
+Provide a string to `@label` to render the text into the `<label>` of the Field.
+
+## Hint
+
+Provide a string to `@hint` to render the text into the Hint section of the Field. This is optional.
+
+## Error
+
+Provide a string to `@error` to render the text into the Error section of the Field. This is optional.
+
+## Value and onChange
+
+To tie into the input event, provide `@onChange`. `@onChange` will return three arguments:
+
+1. the checked attribute from the target
+2. the raw event object
+3. the indeterminate attribute from the target
+
+It's most common to use this in combination with `@value` which will set the `checked` attribute for the Checkbox.
+
+```hbs
+<Form::CheckboxField
+  @label='Label'
+  @value={{this.value}}
+  @onChange={{this.handleChange}}
+/>
+```
+
+```js
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class extends Component {
+  @tracked value;
+
+  @action
+  handleChange(value, e, indeterminate) {
+    console.log({ e, indeterminate, value });
+    this.value = value;
+  }
+}
+```
+
+## Disabled State
+
+Set the `@isDisabled` argument to disable the checkbox.
+
+## Indeterminate State
+
+Set the `@isIndeterminate` argument. To learn more, view the documentation Checkbox documentation.
```

Comment:
```suggestion
Set the `@isIndeterminate` argument. To learn more, view the Checkbox documentation.
```

Could we also have it be a hyperlink to that documentation?

Reply:
Good idea! Added!